<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingForm</class>
 <widget class="QWidget" name="TrainingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>374</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Global network and training configuration:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>374</width>
     <height>116</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="m_CBTransferFunction">
       <item>
        <property name="text">
         <string>tanh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>log</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>binary</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Maximum number of training cycles:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum error delta (x10&lt;span style=&quot; vertical-align:super;&quot;&gt;-4&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Network's transfer function:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="m_SBMax">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_SBError">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
